Auth0/String+URLSafe.swift

```diff
@@ -0,0 +1,35 @@
+// String+URLSafe.swift
```

Comment:
Extension to decode data encoded in Base64Url.

Reply:
It's public because it's the counterpart of the already-public `a0_encodeBase64URLSafe()`.